server/frontend/app.py

```diff
@@ -0,0 +1,28 @@
+import os
+from flask import Flask, render_template, jsonify
+from flask_cors import CORS
+
+app = Flask(__name__, static_folder='static', static_url_path='/')
+# allow CORS for all domains on all routes
+CORS(app)
+
+
+# Hello world index
+@app.route('/')
+def hello_world():
+    return render_template('index.html')
+
+@app.route('/legacy')
+def legacy():
+    # Render the legacy.html
+    return render_template('legacy.html')
+
+
+# return a json error if the send a POST to /solve
+@app.route('/solve', methods=['POST'])
+def solve():
+    return jsonify({'error': 'This is a legacy endpoint. Please see solver.planning.domains for updated usage.'}), 400
+
+
+if __name__ == "__main__":
+    app.run("0.0.0.0", port=5001, debug=True)
```

Comment:
Should this be port=80? Otherwise it will be listening to the same port as the server/api/app.py. In the [frontend/dockerfile](https://github.com/AI-Planning/planning-as-a-service/pull/50/files#diff-9ffd608ca67b116dac61423f7179af42d443c255626a908f09408111c92031cfR23) we are exposing gunicorn with port 80.

Reply:
Ah, you're right. I think it still works because of the docker line that invokes it...

```
CMD ["gunicorn", "--bind", "0.0.0.0:80", "app:app"]
```

...but it's a bit confusing and unnecessary to have it sit at 5001 if you run app.py directly. Will fix, thanks!